jvector-base/src/main/java/io/github/jbellis/jvector/pq/CompressedVectors.java

```diff
@@ -1,136 +1,14 @@
-/*
- * Copyright DataStax, Inc.
```

Comment:
Did you mean to remove the header?

Reply:
No. Probably happened w/ automated extract-interface refactor. Will fix.